### PR TITLE
docs: modify ts branch setup in readme

### PR DESCRIPTION
**Pull request by jaxzhou**
## Why

ts branch has some differences with js branch.  even the ts branch is on building, we still need to update the readme to make sure it can be setup .

### CODE REVIEW

**Comment by joncodo** [comment]
With a post request like this:  

```
POST  /sources/${source id}
{
    collector: ['Issue'],
    enricher: ['LeadTime'],
    options: {
        projects: ['ProjectName'],
        boards: ['Scrum Board Id']
    }
}

response:
{ task: 'task id' }
```

How does a user set their jira auth tokens?

**Comment by joncodo** [comment]
I also don't see anything on data base migrations. If I make a new collector, where does the data get stored in the code?

**Comment by narrowizard** [comment]
> How does a user set their jira auth tokens?
We should create jira source first, with the request as following
```
POST /sources
{
    type: 'Jira',
    options: {
        host: '',
        username: '',
        password: 'password or api token'
    }
}
```
The jira auth info will be stored with the source.

> I also don't see anything on data base migrations. If I make a new collector, where does the data get stored in the code?

Seems database layer is not ready yet.

File: README.md [hunk 1]
```diff
@@ -36,34 +48,61 @@ Additionally to use the postgres database as data source inside grafana, ensure
...
-1. From the terminal, execute `npm run all`
-2. From Postman (or similar), send a request like...
+```
+POST  /sources
+{
+    type: 'Jira',
+    options: {
+        host: '',
+        username: '',
+        password: 'password or api token'
```
[the remark this answers is not in the document]
Comment (jaxzhou) [reply README.md:60]: it'a api description. not the key point of this pr.

**Pull request merged**